book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- You have not implemented the `has-many` relationship between the `book` class and the `rental` class. Please implement a method add_rental for this purpose 🙏.

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```